constants.js

```diff
@@ -22,3 +22,17 @@ export const STORY_TYPES = ['feature', 'bug', 'chore', 'release'];
 export const UNESTIMATED_STORY_TYPES = ['bug', 'chore'];
 
 export const ESTIMATE_NOT_REQUIRED_STATES = ['unstarted', 'unscheduled'];
+
+export const STORY_REVIEW_STATUS = ['unstarted', 'in_review', 'pass', 'revise'];
+
+export const STORY_REVIEW_TYPES = {
```

Comment:
Do these change at all depending on project? We may need to fetch it from the project (see what i'm doing for labels)

Reply:
It surely do. I hardcoded this for fast testing purposes, mainly because I could not find an endpoint to retrieve project types. I will check your solution!

Reply:
Update about this: review types are now fetched from API, but Pivotal only provides an array of ids, without further information. Not sure how we can show in the frontend what type of review it is.

Reply:
@RamiroPinol I *think* you can use () in the query to get subtypes, checkout what I did for comments.

Reply:
Here's the link: https://github.com/opticpower/tracker/blob/master/handlers/PivotalHandler.ts#L9

Reply:
@mattvv Unfortunately that is not supported: `"Specifying sub-tree for 'review_type_ids' is not supported"`